README.md

```diff
@@ -28,6 +28,7 @@ For releases and release notes, see the [Releases](https://github.com/yetanalyti
 - [Authority Configuration](doc/authority.md)
 - [Docker Image](doc/docker.md)
 - [OpenID Connect Support](doc/oidc.md)
+  - [Auth0 Setup Guide](doc/oidc/auth0.md)
```

Comment:
Extra indent?

Reply:
Ah I didn't realize it was intentional. Probably should've, since the auth0 doc is in the oidc directory.

Which brings me to a question: is this anticipating future docs for OIDC IDPs _other_ than auth0?

Reply:
Yeah I'm now wondering if the current Keycloak demo section on the main OIDC page can be fleshed out into its own page in that directory.

Reply:
I guess it could, but it's not really a guide per se, just "here's a demo"